ros2_ws/Dockerfile.nvidia

```diff
@@ -0,0 +1,285 @@
+ARG NVIDIA_BASE_IMAGE=nvcr.io/nvidia/pytorch
+ARG NVIDIA_BASE_TAG=25.06-py3
```

Comment:
this is the latest version ubuntu 24.04 and python 3.12 with very recent cuda versions as well. I think 24.12 might be a bit more fitting for 22.04 devices. I will have to verify, but ultimately taking a small step back in this version is not going to be harmful.

For our humble images we'd have to go further back to ensure the necessary python and cuda version I believe